src/components/Rockets/Rocket.js

```diff
@@ -1,27 +1,42 @@
 import React from 'react';
-import './Rocket.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+// mport './Rocket.css';
```

Comment:
- Please make your code base clean by deleting commented codes, leaving unused code in your codebase can sometimes violate the "clean code" principle, which emphasizes the importance of writing code that is easy to read, understand, and maintain. 
- Clean code is easy to modify and update and is free of unnecessary complexity, redundancy, and clutter.